camisole/languages/rust.py

```diff
@@ -3,5 +3,5 @@
 
 class Rust(Lang):
     source_ext = '.rs'
-    compiler = Program('rustc', opts=['-W', 'warnings', '-O'])
+    compiler = Program('rustc', opts=['-W', 'warnings', '-C", "opt-level=3'])
```

Comment:
You've used double quotes in your edit but not in the original change, this will not parse correctly

Reply:
Sorry 😅